pandas/core/groupby/grouper.py

```diff
@@ -442,7 +442,7 @@ def get_grouper(
     observed: bool = False,
     mutated: bool = False,
     validate: bool = True,
-) -> Tuple[BaseGrouper, List[Hashable], FrameOrSeries]:
+) -> "Tuple[ops.BaseGrouper, List[Hashable], FrameOrSeries]":
```

Comment:
any reason to add quotes? don't we lose the black formatting checking by doing this?

Reply:
Without the quotes we get a circular import error, so needs to be quoted.